go.mod

```diff
@@ -1,3 +1,11 @@
 module github.com/cpanato/action-mattermost-notify
 
-go 1.12
+go 1.13
+
+require (
+	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200114132950-e77ef9733c97
```

Comment:
How did you generate this line? I was expecting it to look like https://github.com/mattermost/mattermost-plugin-jira/blob/master/go.mod#L11